### PR TITLE
Depl-171756 Check service/ps-web installed version in the update-ov.sh and do not reinstall if the same version already deployed

### PR DESCRIPTION
**Pull request by KupchinskiyVasiliy**

### CODE REVIEW

**Review by asmoiseenko — APPROVED** [review]

**Review by asmoiseenko — COMMENTED** [review]
what about snapshots? We should always redeploy snapshot versions

**Comment by KupchinskiyVasiliy** [comment]
> what about snapshots? We should always redeploy snapshot versions

Done.

File: utils.sh [hunk 1]
```diff
@@ -117,11 +117,22 @@ function extract_and_read_artifact_version() {
...
 
     grep 'Implementation-Version' "$MANIFEST_PATH" | cut -d ' ' -f2 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//'
 }
 
+function is_snapshot_version() {
+    local VERSION
+    VERSION="$1"
+
+    if [[ "$VERSION" == *-SNAPSHOT ]]; then
+        return 0
```
Comment (asmoiseenko) [inline utils.sh:130]: you return `0` and `1` but evaluate as a boolean `if ! is_snapshot_version "$NEW_VERSION"`

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: utils.sh (hunk 1 above)
Comment (KupchinskiyVasiliy) [reply utils.sh:130]: Yes, this is correct. Function return some exit code using `return` and then `if command; then` or `if ! command; then` check if command returned success exit code (0) - true or not-success (!=0) - false

Comment (KupchinskiyVasiliy) [reply utils.sh]: Simplifed to 
```
function is_snapshot_version() {
    local VERSION
    VERSION="$1"

    [[ "$VERSION" == *-SNAPSHOT ]]
}
```

Comment (asmoiseenko) [reply utils.sh]: and you do need a `return` statement in this simplified version? Confusing

Comment (KupchinskiyVasiliy) [reply utils.sh]: No need in simplified version. This is also possible variant of return exit code.

**Review by asmoiseenko — APPROVED** [review]

**Pull request merged**